Dockerfile

```diff
@@ -4,9 +4,8 @@ FROM plone/frontend-builder:${VOLTO_VERSION} as builder
 
 # Build Volto Project and then remove directories not needed for production
 RUN <<EOT
-    set -e 
-    yarn build
-    rm -rf cache omelette .yarn/cache
+    set -e
+    pnpm build
 EOT
```

Comment:
This whole thing should simply become `RUN pnpm build`. No heredoc syntax needed since it's now a single command.